### ISSUE TITLE
Block Alignment for Add to Cart Button Too Long

### ISSUE THREAD

**Issue by black-eye** [pojome/elementor#6749]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
After updating to one of the last versions of EPro, the Block (Justify) Alignment for Add to Cart Button is too long:

![obrazek](https://user-images.githubusercontent.com/5634870/51222461-215e3680-193e-11e9-8b0b-81b4e553e1bf.png)

It used to work fine, unfortunately I cannot be more specific than the last version it worked correctly was EPro 2.1.10.

## Steps to reproduce
Just align Add to Cart Button to Justified (see the picture).

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: (Debian) v5.6.39-84.1
	PHP Version: 7.0.33
	PHP Max Input Vars: 5000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.9.9
	Site URL: https://www.bioapoteka.cz
	Home URL: https://www.bioapoteka.cz
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: cs-CZ
	Timezone: Europe/Prague
	Debug Mode: Inactive

== Theme ==
	Name: Bio Apotéka WPBF
	Version: 1.0.0
	Author: Petr Jandák
	Child Theme: Yes
	Parent Theme Name: Page Builder Framework
	Parent Theme Version: 1.13.1
	Parent Theme Author: David Vongries

== User ==
	Role: administrator
	WP Profile lang: cs_CZ
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:64.0) Gecko/20100101 Firefox/64.0

== Active Plugins ==
	Activity Log
		Version: 2.5.2
		Author: Activity Log Team

	Admin Menu Editor
		Version: 1.8.5
		Author: Janis Elsts

	Advanced Custom Fields PRO
		Version: 5.7.9
		Author: Elliot Condon

	All-in-One WP Migration
		Version: 6.77
		Author: ServMask

	Central Color Palette
		Version: 1.13.1
		Author: Gáravo

	Checkout Field Editor for WooCommerce
		Version: 1.3.0
		Author: ThemeHiGH

	Clean Image Filenames
		Version: 1.2.1
		Author: Upperdog

	Cookie Notice
		Version: 1.2.45
		Author: dFactory

	Customizer Export/Import
		Version: 0.9
		Author: The Beaver Builder Team

	Custom Order Numbers for WooCommerce
		Version: 1.2.5
		Author: Tyche Softwares

	Disable Comments
		Version: 1.8.0
		Author: Samir Shah

	Duplicate Post
		Version: 3.2.2
		Author: Enrico Battocchi

	Elementor
		Version: 2.4.1
		Author: Elementor.com

	Elementor Extras
		Version: 2.0.8
		Author: Namogo

	Elementor Pro
		Version: 2.3.1
		Author: Elementor.com

	Google Analytics Dashboard for WP (GADWP)
		Version: 5.3.7
		Author: ExactMetrics

	Intuitive Custom Post Order
		Version: 3.1.1
		Author: hijiri

	iThemes Security
		Version: 7.2.0
		Author: iThemes

	Nested Pages
		Version: 3.0.7
		Author: Kyle Phillips

	Page Builder Framework Premium Addon
		Version: 1.11
		Author: MapSteps

	Perfect WooCommerce Brands
		Version: 1.7.0
		Author: Alberto de Vera Sevilla

	ShortPixel Image Optimizer
		Version: 4.12.5
		Author: ShortPixel

	SVG Support
		Version: 2.3.15
		Author: Benbodhi

	The SEO Framework
		Version: 3.2.1
		Author: Sybre Waaijer

	TinyMCE Advanced
		Version: 4.8.0
		Author: Andrew Ozz

	Ultimate Addons for Elementor
		Version: 1.8.4
		Author: Brainstorm Force

	User Profile Picture
		Version: 2.1.1
		Author: Ronald Huereca

	WooCommerce
		Version: 3.5.3
		Author: Automattic

	Woo Doprava
		Version: 1.5.9
		Author: Vladislav Musílek

	Woo GoPay Inline
		Version: 1.6.5
		Author: Vladislav Musílek

	WP Media folder
		Version: 4.7.5
		Author: Joomunited

	Zalomení
		Version: 1.4.7
		Author: Honza Skypala


== Log ==
	: 
Log: showing 8 of 82019-01-16 02:50:58 [info] elementor::elementor_updater Started 
2019-01-16 02:50:58 [info] Elementor/Upgrades - _v_2_3_0_widget_image Start  
2019-01-16 02:50:58 [info] Elementor/Upgrades - _v_2_3_0_widget_image Finished 
2019-01-16 02:50:58 [info] Elementor/Upgrades - _v_2_3_0_template_type Start  
2019-01-16 02:50:58 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '2.2.5',
  'to' => '2.4.1',
)]
2019-01-16 02:50:58 [info] Elementor/Upgrades - _v_2_3_0_template_type Finished 
2019-01-16 02:50:58 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.2.5',
  'to' => '2.4.1',
)]
2019-01-16 02:50:59 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '2.2.5',
  'to' => '2.4.1',
)]

PHP: showing 12 of 12PHP: 2019-01-16 02:51:12 [warning X 1][/www/doc/www.bioapoteka.cz/www/wp-admin/includes/plugin.php::254] opendir(/www/doc/www.bioapoteka.cz/www/wp-content/plugins/the-seo-framework-extension-manager): failed to open dir: No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 02:54:11 [warning X 1][/www/doc/www.bioapoteka.cz/www/wp-admin/includes/class-wp-automatic-updater.php::98] is_dir(): open_basedir restriction in effect. File(/www/.bzr) is not within the allowed path(s): (/www/doc/www.bioapoteka.cz:/disk2/doc/www.bioapoteka.cz:/usr/share/php:/www/tmp:/tmp) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 02:54:16 [warning X 3][/www/doc/www.bioapoteka.cz/www/wp-includes/class-phpass.php::68] is_readable(): open_basedir restriction in effect. File(/dev/urandom) is not within the allowed path(s): (/www/doc/www.bioapoteka.cz:/disk2/doc/www.bioapoteka.cz:/usr/share/php:/www/tmp:/tmp) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 02:54:58 [warning X 1][/www/doc/www.bioapoteka.cz/www/wp-admin/includes/file.php::168] closedir() expects parameter 1 to be resource, boolean given [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 02:59:18 [notice X 2][/www/doc/www.bioapoteka.cz/www/wp-content/themes/bioapoteka-wpbf/functions.php::7] Use of undefined constant WPBF_CHILD_THEME_DIR - assumed 'WPBF_CHILD_THEME_DIR' [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 03:04:58 [warning X 1][/www/doc/www.bioapoteka.cz/www/wp-includes/plugin.php::658] strpos(): Empty needle [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 03:07:54 [warning X 1][/www/doc/www.bioapoteka.cz/www/wp-content/plugins/shortpixel-image-optimiser/class/wp-short-pixel.php::3337] unlink(/wpbf-premium@2x.zip): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 03:32:02 [warning X 1][/www/doc/www.bioapoteka.cz/www/wp-content/plugins/shortpixel-image-optimiser/shortpixel_api.php::792] unlink(/tmp/ade53cde9c7d19a69f9f7d4f4d5edd50-lossy-Px3k31.tmp): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 03:32:09 [warning X 1][/www/doc/www.bioapoteka.cz/www/wp-content/plugins/shortpixel-image-optimiser/shortpixel_api.php::792] unlink(/tmp/4344cf49e1f95637c36633995777c5f5-lossy-9VIIun.tmp): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 03:32:16 [warning X 1][/www/doc/www.bioapoteka.cz/www/wp-content/plugins/shortpixel-image-optimiser/shortpixel_api.php::792] unlink(/tmp/15bd8c6c059a5ee16ec411aa1a102c71-lossy-wEv7X2.tmp): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 03:32:23 [warning X 1][/www/doc/www.bioapoteka.cz/www/wp-content/plugins/shortpixel-image-optimiser/shortpixel_api.php::792] unlink(/tmp/5abb1f4ed204599226f3dd405f0bd596-lossy-8gDDyo.tmp): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-16 03:32:29 [warning X 1][/www/doc/www.bioapoteka.cz/www/wp-content/plugins/shortpixel-image-optimiser/shortpixel_api.php::792] unlink(/tmp/b1f2412fc214a80fd1d8a92ef160381b-lossy-56S53E.tmp): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				
```
</details>

**Comment by shilo-ey** [comment]
@black-eye 
This feature has been added to the Elementor Pro v2.4.1, feel free to upgrade.

Thanks

**Comment by black-eye** [comment]
@shilo-ey I'm affraid we are running in circles here. One problem solved, another occures.
Now, when justify alignment is selected on desktop, all the lower breakpoints are justified too, with no possibility of change.

![obrazek](https://user-images.githubusercontent.com/5634870/51688703-70b6fd80-1ff5-11e9-9c13-41b6237d8dd4.png)

Seems like the problem is similar to another one once solved: #5830.
